### PR TITLE
Added documentation for max image size

### PR DESCRIPTION
**Pull request by zburstein**
Add documentation for feature https://github.com/anchore/anchore-engine/issues/786

Signed-off-by: Zane Burstein <zane.burstein@anchore.com>

### CODE REVIEW

**Comment by zhill** [comment]
Updated PR base to merge into 3.0-dev instead of master

File: content/docs/using/cli_usage/images/_index.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ For an image that has not yet been analyzed, the status will appear as *not_anal
 
 The image type is shown as `docker`, future release will support the analysis of OCI formatted images.
 
+As of v3.0, Anchore Enterprise can be configured to have a size limit for images being added for analysis. This is disabled by default, but can be configured through the configuration variable `max_compressed_image_size_mb`, which if using the default configuration file can be set through `ANCHORE_MAX_COMPRESSED_IMAGE_SIZE_MB` env variable. This defines a size limit (MB) of the compressed size of the image and will prevent the image from being added for analysis in anchore if the image exceeds the given size. A value less than 0 disables this functionality and allows an image of any size to be added. Note that this size limit is also applied to [tag subscriptions]({{< ref "docs/using/cli_usage/subscriptions/_index.md#tag-updates" >}}) and [repository watchers]({{< ref "/docs/using/cli_usage/repositories/_index.md#watching-repositories" >}}).
```
Comment (zhill) [inline content/docs/using/cli_usage/images/_index.md:39]: Should the "value less than 0 disables" be "less than or equal to 0"?

Comment (zburstein) [reply content/docs/using/cli_usage/images/_index.md:39]: No its set up so negative values disable the feature

**Comment by hev** [comment]
@Btodhunter I feel like these docs needs details and support in the helm chart.

**Comment by zburstein** [comment]
@hev Brady added it to the chart for 3.0 (https://github.com/anchore/anchore-charts/commit/55603fffb48457e6186b592e67eaef61088cbe45) and I have updated this PR with instructions on how to add it to helm

File: content/docs/using/cli_usage/images/_index.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ For an image that has not yet been analyzed, the status will appear as *not_anal
...
 
+As of v3.0, Anchore Enterprise can be configured to have a size limit for images being added for analysis. Images that exceed the configured maximum size will not be added to Anchore and the catalog service will log a message. This size limit is also applied to [tag subscriptions]({{< ref "docs/using/cli_usage/subscriptions/_index.md#tag-updates" >}}) and [repository watchers]({{< ref "/docs/using/cli_usage/repositories/_index.md#watching-repositories" >}}) in addition to the standard API for adding images to Anchore for analysis. 
+
+The max size feature is disabled by default but can be enabled through the `max_compressed_image_size_mb` configuration variable, which represents the size limit in MB of the compressed image. Values less than 0 will disable the feature and allow images of any size to be added to Anchore and non integer values will cause bootstrap of the service to fail. If using compose with the default config, this  can be set through the `ANCHORE_MAX_COMPRESSED_IMAGE_SIZE_MB` env variable on the catalog service. If using helm, it can be defined in the values file via `anchoreGlobal.maxCompressedImageSizeMB`
```
Comment (dspalmer99) [inline content/docs/using/cli_usage/images/_index.md:41]: Will a 0 value be enforced and prevent users from adding any images?

Comment (dspalmer99) [inline content/docs/using/cli_usage/images/_index.md:41]: Nit: There's a double space in "this  can"

Comment (dspalmer99) [reply content/docs/using/cli_usage/images/_index.md:41]: Nit: "non-integer", not "non integer"

**Review by dspalmer99 — CHANGES_REQUESTED** [review]
Please specify what configuring a 0 value does. Otherwise looks good to me outside of a few minor grammatical nits.

File: content/docs/using/cli_usage/images/_index.md (hunk 2 above)
Comment (zburstein) [reply content/docs/using/cli_usage/images/_index.md:41]: Yeah it will. I can specify that

Comment (zburstein) [reply content/docs/using/cli_usage/images/_index.md]: @dspalmer99 Updated

**Review by dspalmer99 — APPROVED** [review]

**Pull request merged**